node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
For what it is worth, I am not aware of anyone actively maintaining this plugin. So, unless another Light Table maintainer says otherwise, I'll leave it up to you.